### DISCUSSION TITLE
Expand scope beyond Mastodon

### DISCUSSION THREAD

**Discussion by anantshri** [adamghill/fediview#15] [Ideas]
Currently the fediview seems to be strictly focused on mastodon but the underlying protocol is used by a lot of other softwares. Is there any plans to expand the scope.

For example : I use gotosocial. the current version is 0.8.1 and that is the first check that it fails and cant proceed although the software is near mastodon api compatible.

error message recieved `URL might be invalid: Version check failed (Need version 1.0.0)`

**Comment by adamghill** [comment]
I'd love to support other fediverse clients in addition to mastodon, but I probably won't have time to add it myself (or handle testing/maintaining). I'd appreciate a PR to add support. 👍
